neo/rawio/axonarawio.py

```diff
@@ -66,11 +66,28 @@ class AxonaRawIO(BaseRawIO):
     def __init__(self, filename):
         BaseRawIO.__init__(self)
 
-        # We accept base filenames, .bin and .set extensions
-        self.filename = filename.replace('.bin', '').replace('.set', '')
-        self.bin_file = os.path.join(self.filename + '.bin')
-        self.set_file = os.path.join(self.filename + '.set')
-        self.set_file_encoding = 'cp1252'
+        # Accepting filename with arbitrary suffix as input
+        self.filename = pathlib.Path(filename).with_suffix('')
```

Comment:
I am not sure if this restriction is necessary, since any filename is accepted from the user side and we only internally use the basename as reference. Artificially restricting this to not accept the basename in the first place feels a bit too artificially restrictive to me...

Reply:
Ok for me to keep it as is then 🙂